tools/idf.py

```diff
@@ -635,10 +635,16 @@ def parse_project_dir(project_dir):
     all_actions = {}
     # Load extensions from components dir
     idf_py_extensions_path = os.path.join(os.environ["IDF_PATH"], "tools", "idf_py_actions")
-    extra_paths = os.environ.get("IDF_EXTRA_ACTIONS_PATH", "").split(';')
-    extension_dirs = [idf_py_extensions_path] + extra_paths
-    extensions = {}
+    # Make a set to automatically handle duplicate paths
```

Comment:
This comment doesn't make sense anymore,
and could you remove it and squish all commits into one?

Reply:
fixed and squashed